src/useStyles.js

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = (borderStyle) => makeStyles((theme) => {
+  const containerBorderStyle = (borderStyle.size && borderStyle.type) && {
+    border: `${borderStyle.size}px ${borderStyle.type} ${borderStyle.color ? borderStyle.color : 'black'}`,
+    'border-radius': `${borderStyle.radius > 0 ? borderStyle.radius : 0}px`,
+  }
+
+  const tableBorderRadius = borderStyle.radius > 0 && {
```

Comment:
border-radius is still in effect when there is a toolbar on top of the column heads:

![Screen Shot 2021-04-12 at 10 26 31 AM](https://user-images.githubusercontent.com/59836949/114410817-aa926100-9b79-11eb-88bb-225f7f696170.png)

Reply:
just fixed the problem and added isBorder prop for the default values. Let me know what you think.